grading/load_config_json.cpp

```diff
@@ -1404,3 +1419,28 @@ void VerifyGraderDeductions(nlohmann::json &json_graders) {
     std::cout << "ERROR! DEDUCTION SUM < 1.0: " << sum << std::endl;
   }
 }
+
+std::vector<std::string> gatherAllTestcaseIds(const nlohmann::json& complete_config){
```

Comment:
similar logic to function "find_testcase_by_id"
Just wondering if they should use the same helper function?